packages/strapi-plugin-upload/admin/src/components/SortPicker/SortButton.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { themePropTypes } from 'strapi-helper-plugin';
+
+const SortButton = styled.button`
+  height: 32px;
+  padding: 0 10px;
+  line-height: 30px;
+  background-color: ${({ theme }) => theme.main.colors.filters.background};
+  border: 1px solid ${({ theme }) => theme.main.colors.filters.border};
+  color: ${({ theme }) => theme.main.colors.filters.color};
+  font-weight: ${({ theme }) => theme.main.fontWeights.semiBold};
```

Comment:
If you use the `Text` component you don't have to set the `font-size` and `font-weight`

Reply:
I would prefer the content to be a <span /> and not a <p /> because the behavior with the icon svg is naturally closer than the one we want. And I think the text-style defined here is not a problem because we want the button to have this style so it will be also skinned if we set a plain text as children.

Reply:
You can use the component the `as` prop to change the type.

Reply:
You can use it as follows:

```
const SortButton = styled(props => <Text as="button" fontWeight="semiBold" />)``
```